packages/css-library/src/stylesheets/elements.scss

```diff
@@ -5,3 +5,145 @@
 // p {
 //    @include @usa-paragraph
 // }
+
+@use "../../dist/tokens/scss/variables" as *;
+
+$color-link-default-hover: rgba($color-black, 0.05);
+
+/* 
+ Styles that are specific to VA.gov using CSS type selectors (elements/tags).
+*/
+
+.feature {
+  background: $color-primary-alt-lightest;
+  padding: 1em;
+  clear: both;
+  margin: 1.5em 0 1.5em 0;
+
+  p {
+    color: inherit;
+    font: inherit;
+    margin-bottom: 0;
+    margin-top: 0;
+    padding-bottom: 0.5em;
+  }
+
+  h2 {
+    margin-top: 0;
+  }
+
+  h3 {
+    margin-top: 0;
+  }
+  h4 {
+    margin-top: 0;
+  }
+  ul {
+    margin: 0 0 0.5em 1.5em;
+    padding: 0;
+    li {
+      list-style: square;
+      margin: 0;
+    }
+  }
+}
+
+.va-quicklinks {
+  ul {
+    margin-top: 12.8px; // Original 0.8rem to 16px base font
+  }
+  li {
+    padding: 8.96px 0; // Original 0.56rem to 16px base font
+  }
+  h3 {
+    line-height: 1.2em;
+    font-size: 1.65em;
+    color: $color-primary-darker;
+    margin: 0 0 16px 0; // Original 1rem to 16px base font
+    padding: 0 0 0.2em 0;
+  }
+}
+
+// Playbook and Facility Locator
+// Used in content-build: src/site/includes/breadcrumbs-playbook.html
+
+.va-nav-breadcrumbs--playbook {
+  background: none;
+  margin: 0 0 1em 0;
+  padding: 0;
+
+  h2 {
+    font-size: 1em;
+  }
+
+  li {
+    text-transform: none;
+    margin: 0.25em 0;
+    padding: 0.1em;
+    display: inline-block;
+    line-height: 1.15em;
+    vertical-align: middle;
+    font-size: 1.35em;
+
+    &:hover {
+      border-bottom: none;
+      text-decoration: none;
+    }
+  }
+
+  a {
+    border-bottom: 3px solid $color-white;
+    text-decoration: none;
+    color: $color-white;
+    &:hover {
+      text-decoration: none;
+      background: $color-link-default-hover;
+      border-bottom: 3px solid $color-gold;
+    }
+  }
+}
+
+// Notice / feedback banners
+
+.va-notice--banner {
+  p {
+    margin: 0;
+  }
+  a {
+    background-image: none;
+    color: inherit;
+    padding: 0;
+  }
+}
+
+// Info cards
+
+.card {
```

Comment:
Could this be replaced with `va-card`? I see this class in both vets-website and content-build:

<img width="633" alt="Screenshot 2023-12-11 at 3 40 42 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/c3db4890-f95b-4dec-8291-24e537fa9bcc">

<img width="455" alt="Screenshot 2023-12-11 at 3 40 52 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/61b64635-a7fa-45ed-b359-00b0cdd5ebf8">

Reply:
+1 from me. Worth looking into in a separate ticket.

Reply:
I will remove it from this PR for now then and similar to `.feature`, if we find that it's absolutely a global CSS class that we need to keep, it can be added back to the CSS Library then.